test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/KeepAliveTimeoutTests.cs

```diff
@@ -91,23 +91,27 @@ public class KeepAliveTimeoutTests
         {
             using (var connection = new TestConnection(server.Port))
             {
-                for (var i = 0; i < 10; i++)
+                for (var i = 0; i < 11; i++)
```

Comment:
@pakrym Ha, that's not necessary. I had it in my head that the keep-alive timeout I was using for the tests was 30s, but it's 10s. With the short delay being 3s I'd need 11 iterations to make sure I was getting past the timeout.